package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@signavio/i18n",
-  "version": "2.1.2",
+  "version": "3.0.0",
```

Comment:
1. Is the version 3.0.0 already published? Can't find it
2. Should we also at a deprecation warning to the github readme?

Reply:
I only use this PR to fast forward the new version to the publish on master
I don't believe it make sense to reference the old package at all, every is said on the npm registry page.